packages/lit-dev-tools-cjs/src/api-docs/transformer.ts

```diff
@@ -412,6 +412,64 @@ export class ApiDocsTransformer {
     }
   }
 
+  /**
+   * Remove fields that we don't need for rendering. This makes reading diffs
+   * much easier, since we check the generated JSON file in.
+   */
+  private prunePageData(node: unknown) {
+    if (node instanceof Array) {
```

Comment:
Non actionable question. This vs something like `Array.isArray`?

Reply:
I guess `instanceof Array` has problems when crossing iframe boundaries? But not really relevant here.